src/commands/raid-leaders/common/RaidLeaderCommon.ts

```diff
@@ -1,27 +1,26 @@
 import {
```

Comment:
For the new VC system, it NEEDS to be more clear to raiders which channel is being used, namely from the AFK embed.  I know it says it in the AFK stage, I would include it in the Pre-afk stage as well.

I'm not sure how I feel about it as a whole, but maybe some people will find it useful.  For Dungeoneer, please don't add more than 2 raiding channels.

Reply:
Resolved